src/main/studio/ui/SearchPanel.java

```diff
@@ -160,22 +162,37 @@ private void doSearch(SearchContext context, SearchAction action) {
                     result = SearchEngine.replaceAll(textArea, context);
                 }
             } catch (IndexOutOfBoundsException e) {
-                editorPane.setTemporaryStatus("Error during replacement: " + e.getMessage());
+                editorPane.setTemporaryStatus("Error during replacement: " + e.getMessage(), true);
                 return;
             }
         }
 
         String status;
+        boolean isAlert = false;
         if (! result.wasFound()) {
             status = "Nothing was found";
+            isAlert = true;
```

Comment:
This is just for consistency with the netbeans editor. But I checked a few others:
Notepad++ dings and shows a red error if the text is not found - including if it hits the end of the document and wrap around is not selected.
VSCode displays "No results" in red as opposed to "X of Y" in white.

Reply:
I compared with Sublime text editor which doesn't highlight such situations as error. I think i was such against because of how they are visualised - a lot of bright red (for me it is a sign of dangerous - something dramatic happened), a lot of attention to such mostly normal situation.